lib/node_modules/@stdlib/stats/base/dists/weibull/skewness/manifest.json

```
@@ -0,0 +1,85 @@
+{
+    "options": {
+      "task": "build",
+      "wasm": false
+    },
+    "fields": [
+      {
+        "field": "src",
+        "resolve": true,
+        "relative": true
+      },
+      {
+        "field": "include",
+        "resolve": true,
+        "relative": true
+      },
+      {
+        "field": "libraries",
+        "resolve": false,
+        "relative": false
+      },
+      {
+        "field": "libpath",
+        "resolve": true,
+        "relative": false
+      }
+    ],
+    "confs": [
+      {
+        "task": "build",
+        "wasm": false,
+        "src": ["./src/main.c"],
+        "include": ["./include"],
+        "libraries": [],
+        "libpath": [],
+        "dependencies": [
+          "@stdlib/math/base/assert/is-positive-integer",
+          "@stdlib/math/base/assert/is-finite",
+          "@stdlib/math/base/assert/is-nan",
+          "@stdlib/math/base/special/exp",
+          "@stdlib/constants/float64/ln-two",
+          "@stdlib/math/base/special/ln",
+          "@stdlib/math/base/assert/is-integer",
+          "@stdlib/math/base/special/gamma"
```

```suggestion
          "@stdlib/math/base/assert/is-nan",
          "@stdlib/math/base/special/pow",
          "@stdlib/math/base/special/sqrt",
          "@stdlib/math/base/special/gamma"
```
since most of the header files are not used in the build there is no need to add them here and below for `benchmark` and `examples`